src/proto/single/resp.rs

```diff
@@ -118,6 +120,92 @@ pub async fn read_ok<R: AsyncBufRead + Unpin>(r: R) -> Result<(), Error> {
     Err(bad("server ok", &rr).into())
 }
 
+// ----------------------------------------------
+
+/// Faktory service stats.
```

Comment:
Can we be more descriptive here to distinguish this from `ServerSnapshot`? I _think_ this specifically is statistics related to the job queues (as opposed to the server process)?

Reply:
Exactly, queues and some aggregated data. Updated.